nbgrader/apps/baseapp.py

```diff
@@ -88,13 +171,99 @@ class BaseNbConvertApp(BaseNbGraderApp, NbConvertApp):
     aliases = Dict(nbconvert_aliases)
     flags = Dict(nbconvert_flags)
 
+    use_output_suffix = Bool(False)
+    postprocessor_class = DottedOrNone('')
```

Comment:
Note (to self, also) with `DottedOrNone` you might need to import yourself, see if `Type` would work too and simplify the logic by auto-importing.